src/components/Movie.js

```diff
@@ -8,17 +11,19 @@ const Movie = ({ movie }) => {
   const poster =
     movie.Poster === "N/A" ? DEFAULT_PLACEHOLDER_IMAGE : movie.Poster;
   return (
-    <div className="movie">
+    <Box className="movie" sx={{ width: 500, height: 450, overflowY: 'scroll' }}>
```

Comment:
Please, don't use this styles in line `sx={{ width: 500, height: 450, overflowY: 'scroll' }}`, use your class "movie" to add this styles

Reply:
Understood! It s already fixed.